docs/en/ingest-management/elastic-agent/configuration/generate-standalone-agent-config.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[generate-standalone-agent-config]]
+= Generate a standalone {agent} configuration file
+
+include::{fleet-repo-dir}/standalone-note.asciidoc[]
+
+To get started quickly, generate a standalone configuration in {kib} then
+download it as a starting point for your {agent} configuration. This approach
+saves time, is less error prone, and populates the configuration with a lot of
+details that are tedious to add manually. This process also loads the assets you
+need, such as dashboards and ingest pipelines, before you start your {agent}s.
+
+TIP: If you're a {fleet} user and already have an agent policy you want to
+use in standalone mode, go to *{fleet} > Agents* and click *Add agent*. Follow
+the steps under *Run standalone* to generate the file.
+
+//QUESTION: When using the Integrations UI to create a new agent policy, I saw 
+//I saw the message: "Impossible to create an API key: //feature not enabled
+//exception...Reason: api keys are not enabled."
+//
+//It seems like Kibana _did_ create the policy despite the error, though. I've
+//added the following sentence, but I don't know if it's true or if the error is
+//a bug.
+
+You don't need {fleet} to perform these steps, but API keys must be enabled
+in the {es} configuration (set `xpack.security.authc.api_key.enabled: true`).
+
+. From the main menu in {kib}, click *Add integrations*, and search for the
+{agent} integration you want to use.
+
+. Click the integration to see more details about it, then click
+*Add <Integration>*.
++
+[role="screenshot"]
+image::images/add-integration-standalone.png[Add Nginx integration screen with agent policy selected]
+
+. Under *Configure integration*, enter a name and description for the integration.
+
+. Click the down arrow next to enabled streams and make sure the settings are
+correct for your host.
+
+. For the agent policy, select an existing policy, or click
+*Create agent policy* and create a new one.
++
+{kib} will use this policy to generate the standalone configuration.
+
+. When you’re done, save and continue.
++
+A popup window gives you the option to add {agent} to your hosts.
```

Comment:
I can't answer your questions, but I noticed that you hyphenated `pop-up` twice in this doc, but didn't hyphenate here. Helpful, right? 🤣

Reply:
Haha. Gotta cross the T's and dot the I's.